### PR TITLE
Normalize the config file to minimize noise later

### PR DESCRIPTION
**Pull request by kytrinyx**
We will be scripting a number of changes to the track configuration.
This runs configlet fmt, which ensures that any further changes can be minimized.

See https://github.com/exercism/meta/issues/95

### CODE REVIEW

**Comment by marnen** [comment]
This is harder to read for no apparent gain. What's the reasoning here?

File: config.json [hunk 1]
```diff
@@ -1,63 +1,83 @@
...
-  ],
-  "foregone": [
-
-  ]
-}
+  "foregone": [],
+  "language": "GNU APL",
+  "test_pattern": "\\.tc$",
+  "track_id": "gnu-apl"
+}
```
Comment (marnen) [inline config.json]: No newline at end? That's not great practice [according to the POSIX standard](https://stackoverflow.com/a/729795)...

File: config.json [hunk 2]
```diff
@@ -1,63 +1,83 @@
...
     {
-      "uuid": "05764380-02d5-2a80-0741-3baa1e10b36b61333ec",
-      "slug": "rna-transcription",
       "core": true,
       "difficulty": 1,
-      "topics": ["strings", "arrays", "substitution", "error handling"]
+      "slug": "rna-transcription",
+      "topics": [
+        "arrays",
+        "error_handling",
```
Comment (marnen) [inline config.json]: I thought topic names were meant to be human-readable, so why the underscores? Am I wrong about the human-readability?

**Comment by kytrinyx** [comment]
There's a discussion happening in exercism/meta#95 about how this normalization degrades the readability of the configs.

That discussion has some good suggestions for improvements to the serializer.
I'd appreciate it if you would weigh in on that discussion with any suggestions for improving how we handle configs, consistency, and normalization.

Meanwhile, I'm going to close this and wait for that discussion to play out.

I'm sorry about the noise, and any extra work that I've caused you.

**Comment by marnen** [comment]
You haven't caused me any extra work. I'll look at the other discussion.

**Pull request closed without merging**